pkg/processor/metricfrequencyprocessor/README.md

```diff
@@ -0,0 +1,45 @@
+## <a name="metricfrequencyprocessor"></a>Metric Frequency Processor
```

Comment:
```suggestion
## Metric Frequency Processor
```

I don't believe that's needed unless I'm missing something.

Reply:
This pattern is present in other our processors, so I kept it. Any reason not to have it here?

Reply:
this doc can be linked to with it, going (scrolling) directly to documentation and not github page with files

Reply:
tbf, I see we achieve the same with a link straight to README file instead

so perhaps the authors had something different in mind

Reply:
You don't need the `a` tag for that: 

> README.md#metric-frequency-processor

Reply:
ok, then I don't see reason for this as well, removing